tests/registry_manager_test.py

```diff
@@ -0,0 +1,61 @@
+"""Tests for the kafkit.registry.manager module."""
+
+import os
+from pathlib import Path
+from typing import AsyncGenerator
+
+import pytest
+from aiohttp import ClientSession
+
+from kafkit.registry.aiohttp import RegistryApi
+from kafkit.registry.manager import RecordNameSchemaManager
+
+
+@pytest.fixture(scope="session")
+def schema_root() -> Path:
+    """Pytest fixture for the root path of schemas for testing the
+    RecordNameSchemaManager.
+    """
+    return Path(__file__).parent / "data" / "record_name_schemas"
```

Comment:
This could just be a constant defined at the top of the file rather than a fixture, I think.